examples/kubernetes/connectivity-check/main_tool.cue

```diff
@@ -75,8 +76,17 @@ ccCommand: {
 		}
 	}
 
+	task: filterTraffic: {
+		if #flags.traffic != "any" {
+			resources: [ for x in task.filterName.resources if x.metadata.labels.traffic == #flags.traffic || x.metadata.labels.traffic == "any" {x}]
```

Comment:
I'm not sure I understand why the "any" comparison is necessary, the line towards the top of the file marks `*"any"` as default via the `*`, so it should always have a value..?

Reply:
^ Not a big deal though, I'm happy with the rest of this PR as-is.